pkg/operator/certrotationcontroller/externalloadbalancer.go

```diff
@@ -18,7 +18,6 @@ func (c *CertRotationController) syncExternalLoadBalancerHostnames() error {
 	hostname := infrastructureConfig.Status.APIServerURL
 	hostname = strings.Replace(hostname, "https://", "", 1)
 	hostname = hostname[0:strings.LastIndex(hostname, ":")]
-	hostname = strings.Replace(hostname, "api-int.", "api.", 1)
```

Comment:
I think we need to keep this until we change the installer to set `infrastructureConfig.Status.APIServerURL` to `api`